.travis.yml

```diff
@@ -99,6 +109,9 @@ before_script:
 
 # command to run tests
 script:
+  # Ensures that we can compile all source code files in this Python 3+ version
+  # This behavior is disabled in the 'include' matrix above for Python 2.7
+  - python3 -m compileall -x py2only kolibri -q
```

Comment:
By this point the travis venv has already been activated, so this should just be `python` not `python3` to trigger the matrix set python version. I think this may be why it is passing when we have not fixed this issue yet.

Reply:
Good point -- after opening the PR I didn't have time to wait around for the Travis results.

Will try changing it to `python` and add a temporary `python --version`